unfoldingWord/translationCore
# The main tC screen should be dimmed & disabled when Finder or Explorer is open

#### This pull request addresses:

- The main `tC` screen should be dimmed & disabled when Finder or Explorer is open.


#### How to test this pull request:

- Export a project to `CSV` and/or `USFM` and it should dim the screen not allowing to click the `FAB`


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/3011)
